examples/cmd/snapshotting/Makefile

```diff
@@ -0,0 +1,67 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+RELEASE_URL=https://github.com/firecracker-microvm/firecracker/releases
+VER=v1.0.0
+ARCH=$(shell uname -m)
+PATH  := $(PATH)
+PWD=$(shell pwd)
+FC_TEST_DATA_PATH?=$(PWD)
+
+all: plugins image vmlinux firecracker
+
+plugins: bin/tc-redirect-tap bin/ptp bin/host-local | bin
+
+bin:
+	mkdir -p bin
+
+bin/tc-redirect-tap: bin
+	GO111MODULE=off GOBIN=$(PWD)/bin \
+	go get github.com/awslabs/tc-redirect-tap/cmd/tc-redirect-tap
+
+bin/ptp: bin
+	GO111MODULE=off GOBIN=$(PWD)/bin \
+	go get github.com/containernetworking/plugins/plugins/main/ptp
+
+bin/host-local: bin
+	GO111MODULE=off GOBIN=$(PWD)/bin \
+	go get github.com/containernetworking/plugins/plugins/ipam/host-local
+	
+image:
+	- sudo cp ${FC_TEST_DATA_PATH}/root-drive-with-ssh.img root-drive-with-ssh.img
```

Comment:
This assumes the user is able to escalate root permissions. It is better practice to check permissions and error.